supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -1,6 +1,31 @@
 [[support]]
 = Support
 
+[[developer-preview-guidance]]
+== Developer Preview
+
+Developer Preview features provide early access to upcoming product features in advance of their possible inclusion in a Red Hat product offering, enabling customers to test functionality and provide feedback during the development process. These features might not have any documentation, are subject to change or removal at any time, and testing is limited. Red Hat might provide ways to submit feedback on Developer Preview features without an associated SLA.
+
+When documenting a Developer Preview feature, follow these guidelines:
+
+* Add an admonition labeled IMPORTANT at the beginning of the Developer Preview content and include the template text.
+* Use initial uppercase capitalization, that is, Developer Preview.
+* Never use the phrase "supported as a Developer Preview", and avoid using "support" in Developer Preview descriptions. Instead, use neutral words like "available", "provide", "capability", and so on.
+* When the Developer Preview feature becomes generally available, remove the IMPORTANT admonition from any document that includes content about the feature.
```

Comment:
We might want to say something here about the possibility of it moving into TP status first.